app/clients/cloudwatch/__init__.py

```diff
@@ -0,0 +1,25 @@
+from app.clients import Client, ClientException
+
+
+class CloudwatchClientResponseException(ClientException):
+    """
+    Base Exception for SmsClientsResponses
+    """
+
+    def __init__(self, message):
+        self.message = message
+
+    def __str__(self):
+        return "Message {}".format(self.message)
+
+
+class CloudwatchClient(Client):
```

Comment:
Is there a benefit to having this class? Feels like we could skip and just have `AwsCloudwatchClient` inherit directly from `app.clients.Client`

Reply:
will do